plugins/community/timostamm-protobuf-ts/v2.8.2/buf.plugin.yaml

```diff
@@ -0,0 +1,17 @@
+version: v1
+name: buf.build/community/timostamm-protobuf-ts
+plugin_version: v2.8.2
+source_url: https://github.com/timostamm/protobuf-ts
+description: Generates Protobuf and RPC for TypeScript.
+output_languages:
+  - typescript
+registry:
+  npm:
+    import_style: module
```

Comment:
Iirc this should just work because it's the plugin default, there is a `output_legacy_commonjs` .. but we do not need to concern ourselves with that.